src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -640,6 +679,7 @@ grpc_chttp2_transport::grpc_chttp2_transport(
   }
   settings.mutable_local().SetMaxHeaderListSize(DEFAULT_MAX_HEADER_LIST_SIZE);
   settings.mutable_local().SetAllowTrueBinaryMetadata(true);
+  settings.mutable_local().SetAllowSecurityFrame(true);
```

Comment:
This should be under the control of a channel arg (that for open source should default off)

Reply:
Done.